worker/lib/workers/qemu.ts

```diff
@@ -241,19 +264,19 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 	public async network(configuration: {
 		wired?: { nat: boolean };
 	}): Promise<void> {
-    const bridgeName: string = this.qemuOptions.network.bridgeName;
-    const bridgeAddress: string = this.qemuOptions.network.bridgeAddress;
-    const dnsmasqArgs = [
-      `--interface=${bridgeName}`,
-      `--dhcp-range=${this.qemuOptions.network.dhcpRange}`,
-      '--conf-file',
-      '--bind-interfaces',
-      '--no-daemon',
-      `--dhcp-leasefile=/var/run/qemu-dnsmasq-${bridgeName}.leases`,
-    ];
+		const bridgeName: string = this.qemuOptions.network.bridgeName;
+		const bridgeAddress: string = this.qemuOptions.network.bridgeAddress;
+		const dnsmasqArgs = [
+		`--interface=${bridgeName}`,
+		`--dhcp-range=${this.qemuOptions.network.dhcpRange}`,
+		'--conf-file',
+		'--bind-interfaces',
+		'--no-daemon',
+		`--dhcp-leasefile=/var/run/qemu-dnsmasq-${bridgeName}.leases`,
+		];
```

Comment:
Same as the above comment on diff noise, this should be in a separate commit, which makes things easier to review. :)